README.md

```diff
@@ -122,6 +125,10 @@ If you're using [Emacs](https://www.gnu.org/software/emacs/), you can
 [modify your configs](https://github.com/serokell/universum/issues/8#issuecomment-276444879)
```

Comment:
If you're using [Emacs](https://www.gnu.org/software/emacs/) and don't want to type `import Universum` manually every time, you can [modify your configs](https://github.com/serokell/universum/issues/8#issuecomment-276444879) a little bit.